kobo/apps/subsequences/constants.py

```diff
@@ -9,5 +12,5 @@
 GOOGLE_CACHE_TIMEOUT = 28800  # 8 hours
 
 
-def make_async_cache_key(user_id: int, submission_id: str, xpath: str, source: str):
-    return f'{SUBSEQUENCES_ASYNC_CACHE_KEY}-{user_id}-{submission_id}-{xpath}-{source}'
+def make_async_cache_key(*args: List[Any]) -> str:
```

Comment:
🤔 , any reason why you changed this? 
I think I preferred when what is composing the key was explicit. 
Could you at least leave a comment what we are expecting here? Other devs may not know what it is needed to retrieve the key.

Reply:
Yeah, I agree. This was from previous commits. I'll fix it